ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master.yaml

```diff
@@ -0,0 +1,24 @@
+base_images:
+  operator-test-playbooks:
+    name: operator-test-playbooks
+    namespace: operator_testing
```

Comment:
This does not seem to exist:

```console
$ oc --context api.ci --namespace operator_testing get is operator-test-playbooks --as system:admin
Error from server (NotFound): namespaces "operator_testing" not found
```

Also it does not seem to be used anywhere, so why import it as a base image?

Reply:
Hi @petr-muller,
`make update` deleted `cluster: quay.io` line. Our image is public. Ouch, it wasn't, but it is currently, our fault. Now public. https://quay.io/repository/operator_testing/operator-test-playbooks?tab=tags

Regarding `invalid-owners-file` label. Is there something wrong. People from #forum-dp-platform told me that there is no need to be a member to create PR.

Reply:
It's not supposed to be a quay image, but an image present on the CI cluster. Looks like you may want to add an imagestream here: https://github.com/openshift/release/tree/master/core-services/supplemental-ci-images

`invalid-owners-file` validates the submitted `OWNERS` files. The people in the `OWNERS` must be "trusted" in order to use the features (approvals and reviews). It does not concern creating PRs.